packages/mobile/README.md

```diff
@@ -113,15 +118,16 @@ Install by running the following:
 ```bash
 brew install cask
 brew tap homebrew/cask-versions
-brew cask install homebrew/cask-versions/adoptopenjdk8
+brew install --cask homebrew/cask-versions/adoptopenjdk8
 ```
 
-Alternatively, install Jenv to manage multiple Java versions:
```

Comment:
To my understanding, Jenv isn't really an alternative to installing java 8, just a nice way of managing different versions of Java